tests/integration/test_fetch_orderbook.py

```diff
@@ -40,6 +40,52 @@ def test_get_order_rewards(self):
 
         self.assertIsNone(pd.testing.assert_frame_equal(expected, rewards_df))
 
+    def test_get_batch_rewards(self):
+        block_number = 16735007
+        block_range = BlockRange(block_number, block_number + 50)
+        rewards_df = OrderbookFetcher.get_batch_rewards(block_range)
+
+        expected = pd.DataFrame(
+            {
+                "block_number": [16734995, 16734998],
+                "block_deadline": [16735008, 16735011],
+                "tx_hash": [
+                    "0x203bac6edde8f4dd2e18e7a5e2d81cb721d8b4f1f021217d0d4b55a799efe3f0",
+                    "0xce494850e80b3308d71a5896b7485de0d777af924ebad064d74be4320d027cba",
+                ],
+                "solver": [
+                    "0xde786877a10dbb7eba25a4da65aecf47654f08ab",
+                    "0xde786877a10dbb7eba25a4da65aecf47654f08ab",
+                ],
+                "execution_cost": [7538092113186786.0, 7116851210101934.0],
+                "surplus": [53999674326241.0, 58558783891722.0],
+                "fee": [
+                    0.0,
+                    0.0,
```

Comment:
uh oh...

Reply:
I suspect the DB might contain incorrect values. In the prod DB there are some (old?) entries with nonzero surplus but zero fee.

Reply:
Ok -- so the FEE for these orders is are ZERO. Is it possible they were limit orders and we need to account for this?

Reply:
https://etherscan.io/tx/0x2efb43352a27ba4ce35dd930e39eb4e7ff870aa25ae4b8f2b9b2d67af9b5d76b
is a recent successful settlement where our solver_competition endpoint shows a fee of 6316407881430484, while the production database shows a fee of 0.

Reply:
Checked and both are market orders.

Reply:
<img width="1020" alt="Screenshot 2023-03-13 at 15 58 29" src="https://user-images.githubusercontent.com/11778116/224740420-01f05b7c-3546-4733-98ce-d70658151c02.png">

Def Zero.

Reply:
I will need to manually run the fee calculation against real db to see what happened.

Reply:
I just wanted to remind us that this test is still returning ZERO for the fees... Do we have any idea when or if this will be resolved?

Reply:
I thought this had been fixed. Can you check for a very recent settlement to see if this still ongoing?

E.g., hash `0x6f0d4463061cd86f93979ca8a489f8d5453191d78b24adade196620eb9142c80`

where the fee is `14.442 USDC`

Reply:
Oh yes it's very possible it's been fixed, just not for these records. If there is no reason to expect these records be corrected then I can change the scope of my test.

Reply:
I have found a new (more recent -- like today) block range that captures even more cases. In particular an example where there is no settlement. This is weird now because we are seeing "0x" as the string representing a missing transaction hash. Should probably handle this case in our upload too.

Reply:
Can you share this example without settlement? Just curious to double check what is happening there.

Reply:
My best guess is that the auction was never settled. I think you might know better what actually happened.